joplin/base/migrations/0034_auto_20180927_0731.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.0.8 on 2018-09-27 07:31
```

Comment:
Now that we've copied all the steps from the old model to the new one, we can go ahead and remove the old field/model